Add `(please refer to specific README files below for installation an… (#104)
BioPhoton/rx-angular: README.md

![rx-angular logo](https://raw.githubusercontent.com/BioPhoton/rx-angular/master/images/rx-angular_logo.png)

> **Packages included: (please refer to specific README files below for installation and setup instructions, examples and resources)**

Reviewer: ```suggestion
**Packages included:**
find details in the linked readme files below for installation and setup instructions, examples and resources below
```
